domains/dishant.json

```
@@ -0,0 +1,11 @@
+{
+  "domain":"is-a.dev",
+  "subdomain":"dishant",
+  "owner":{
+    "repo":"https://github.com/Dishant402955/Portfolio",
+    "email":"dishantradadiya3036@gmail.com"
+  },
+  "record":{
+    "CNAME":"https://portfolio-dishants-projects-6881d6c0.vercel.app/"
```

```suggestion
    "CNAME":"portfolio-dishants-projects-6881d6c0.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).